src/Core/Varargs/IVarargsFormatParser.cs

```diff
@@ -21,7 +21,7 @@
 using System.Collections.Generic;
 using Reko.Core.Types;
 
-namespace Reko.Core
+namespace Reko.Core.Varargs
 {
```

Comment:
New 7dec332 commit overwrites this branch with change namespace to `Reko.Core.Analysis`